packages/common/src/extensions/slickCellExternalCopyManager.ts

```diff
@@ -155,15 +155,29 @@ export class SlickCellExternalCopyManager {
       }
 
       // if a custom setter is not defined, we call applyValue of the editor to unserialize
-      if (columnDef.editor) {
+      if (columnDef.editorClass) {
         const tmpDiv = document.createElement('div');
-        const editor = new (columnDef as any).editor({
+        const editor = new (columnDef as any).editorClass({
           container: tmpDiv, // a dummy container
           column: columnDef,
           position: { top: 0, left: 0 },  // a dummy position required by some editors
           grid: this._grid
-        });
+        }) as Editor;
```

Comment:
ahh I missed that earlier, I think we can remove the `any` on the column and make the `editorClass` parsed as `EditorConstructor` that was added not long ago. I'm guessing we might have to explicitly parse it though

```ts
const editor = new (columnDef.editorClass as EditorConstructor)({ })`
```
unless this other suggestion works, which would be better
```ts
const editor = new columnDef.editorClass({ }) as EditorConstructor`
```
Note that there's 2 of these to change

Reply:
not sure if thats gonna work as editorClass can be both the ctor and an editor according to the type

Reply:
yeah ok I'll take a look at that later, I wanted to check on Stackblitz but it looks like it's blocked by the VPN on work laptop, so anyway I can review that on a separate PR if it could be improved (or not)